persistently-store-data-using-Ballerina/backend_server/server.bal

```diff
@@ -0,0 +1,60 @@
+import ballerina/http;
+import ballerina/persist;
+import ballerina/random;
+
+final Client orderDatabase = check new;
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"]
+    }
+}
+service /sales on new http:Listener(9090) {
+    isolated resource function get orders() returns Order[]|error {
```

Comment:
We don't have to add `isolated` explicitly.

Add `isolated` explicitly, resolve the errors, once the code is OK you can remove isolated. Should then get inferred.